back/backend/handlers/admin/show_users.py

```diff
@@ -0,0 +1,89 @@
+from json import loads
+from flask import Blueprint, jsonify, request
+from google.cloud import ndb
+
+from backend.models.wca.person import Person
+from backend.lib import auth
+from backend.models.user import User, Roles
+
+bp = Blueprint('show_users', __name__)
+client = ndb.Client()
+
+
+@bp.route('/get_users')
+def get_users():
+    with client.context():
+        me = auth.user()
+        if not me or not me.HasAnyRole(Roles.AdminRoles()):
+            return jsonify({"error": "Forbidden"}), 403
+
+        # Pagination
+        page = request.args.get('page', 1, type=int)
+        per_page = request.args.get('per_page', 30, type=int)
+        cursor = request.args.get('cursor', None, type=str)
+        if cursor:
+            cursor = ndb.Cursor(urlsafe=cursor)
+        if page < 1:
+            page = 1
+        if per_page not in [10, 20, 30, 40, 50]:
+            per_page = 30
+
+        # Sort
+        sort_field = request.args.get('sort_field', 'name')
+        sort_order = request.args.get('sort_order', 'asc')
+        if sort_field not in ['id', 'name', 'wca_person', 'roles', 'province']:
+            sort_field = 'name'
+        if sort_order.lower() not in ['asc', 'desc']:
+            sort_order = 'asc'
+
+        # Filter
+        filter_text = loads(request.args.get('filter', '')).get("q")
```

Comment:
I believe an empty string will cause `json.loads` to error out:
```suggestion
        filter_text = loads(request.args.get('filter', '{}')).get("q")
```